nodes/clarify_util.js

```diff
@@ -32,39 +32,38 @@ module.exports = {
     return data;
   },
 
-  prepareData: function (RED, msg, config) {
-    let times = null;
-    try {
-      times = RED.util.getMessageProperty(msg, config.signalDataTimes);
-    } catch (e) {}
-
-    let series = null;
-    try {
-      series = RED.util.getMessageProperty(msg, config.signalDataSeries);
-    } catch (e) {}
-
-    if (times === null && series === null) {
+  prepareData: function (RED, msg) {
+    let payload = RED.util.getMessageProperty(msg, 'payload');
+    if (payload === undefined) {
       return null;
     }
 
-    if (!Array.isArray(times)) {
-      throw 'dataTimes must be array';
+    if (typeof payload !== 'object') {
+      throw 'msg.payload must be object';
     }
 
-    if (!Array.isArray(series)) {
-      throw 'dataSeries must be array';
+    if (_.isEmpty(payload)) {
+      throw 'msg.payload can not be empty';
```

Comment:
Actually we should allow messages to not have a payload, right? I.e. to only update meta-data....